src/components/ClusterNodes/NodeDetailsList.tsx

```diff
@@ -372,38 +375,17 @@ export default function NodeDetailsList({ isSuperAdmin, renderRefreshBar, addTab
         setNodeListOffset(0)
     }
 
-    const renderSortDirection = (column: ColumnMetadataType): JSX.Element => {
-        if (column.isSortingAllowed) {
-            if (sortByColumn.value === column.value) {
-                return <span className={`sort-icon ${sortOrder == OrderBy.DESC ? 'desc' : ''} ml-4`} />
-            }
-            return <span className="sort-column dc__opacity-0_5 ml-4" />
-        }
-    }
-
-    const renderNodeListHeader = (column: ColumnMetadataType): JSX.Element => {
-        const nodeColumnClassName = fixedNodeNameColumn
-            ? 'bcn-0 dc__position-sticky  sticky-column dc__border-right'
-            : ''
-        return (
-            <div
-                className={`h-36 list-title dc__inline-block mr-16 pt-8 pb-8 ${
-                    column.label === 'Node' ? `${nodeColumnClassName} w-280 pl-20` : 'w-120'
-                } ${sortByColumn.value === column.value ? 'sort-by' : ''} ${sortOrder === OrderBy.DESC ? 'desc' : ''} ${
-                    column.isSortingAllowed ? ' pointer' : ''
-                } ${column.value === 'status' && 'w-180'}`}
-                onClick={() => {
-                    column.isSortingAllowed && handleSortClick(column)
-                }}
-                data-testid={column.label}
-            >
-                <Tippy className="default-tt" arrow={false} placement="top" content={column.label}>
-                    <span className="dc__inline-block dc__ellipsis-right mw-85px ">{column.label}</span>
-                </Tippy>
-                {renderSortDirection(column)}
-            </div>
-        )
-    }
+    const renderNodeListHeader = (column: ColumnMetadataType): JSX.Element => (
```

Comment:
Suggestion: Instead of creating higher order functions can move the render blocks to a new component

Reply:
Since this PR is going without QA I don't want to make hefty changes; Leaving for future